toplevel/byte/topdirs.mli

```diff
@@ -26,6 +26,9 @@ val dir_use : formatter -> string -> unit
 val dir_use_output : formatter -> string -> unit
```

Comment:
Looks like this file is the same as `toplevel/topdirs.mli`. I guess it should be deleted?

Reply:
Ah yes, that was a stray file, did'nt I push that fix already ?
Thanks

Reply:
Just updated and it's gone now